README

```diff
@@ -30,3 +30,4 @@ Dependencies:
  - python
  - python-jinja2
  - python-fedora
+ - mwclient (via pip)
```

Comment:
If this module is available as RPM, we shouldn't recommend using pip here :)
We may need to update the other package names to reflect their RPM name (which is/was the original intent)